lib/MusicBrainz/Server/Controller/WS/2/Genre.pm

```diff
@@ -7,6 +7,11 @@ use aliased 'MusicBrainz::Server::WebService::WebServiceStash';
 
 my $ws_defs = Data::OptList::mkopt([
     genre => {
+        method   => 'GET',
+        optional => [ qw(fmt limit offset) ],
```

Comment:
So I’m not sure why these options are available.

Reply:
That's a good question! :D Removed.